climpred/bootstrap.py

```diff
@@ -212,11 +212,13 @@ def bootstrap_compute(
     hist=None,
     metric='pearson_r',
     comparison='m2e',
+    dim='init',
```

Comment:
I'm still not fully understanding this. It seems to make things unnecessarily complicated. If I understand correctly from our call, the only options here are 'init' or 'member' that would make sense, the latter of which is for some specific metrics (a few metrics we know use 'member'). So couldn't we just check that the `member` dimension exists if they pick a metric that requires it, and then automatically apply it to that dimension?

Reply:
It can also be list(init,member)